libs/blocks/merch-card/merch-card.js

```diff
@@ -107,6 +107,15 @@ const CARD_TYPES = ['segment', 'special-offers', 'plans', 'catalog', 'product',
 
 const MINI_COMPARE_CHART = 'mini-compare-chart';
 
+// Force cards to refresh once they become visible so that the footer rows are properly aligned.
+const sectionObserver = new MutationObserver((mutations) => {
+  mutations.forEach((mutation) => {
+    if (mutation.type === 'attributes' && mutation.attributeName === 'data-status') {
+      [...mutation.target.querySelectorAll('merch-card')].forEach((card) => card.requestUpdate());
+    }
```

Comment:
Should the MutationObserver be disconnected after executing relevant logic?

Reply:
@overmyheadandbody it is a shared instance for all the `merch-card`s.  
There is not a right moment to call disconnect method as it might break pending observartions.
Since we explicitly check against the `data-status` attribute at the line `113`,  I think it is safe to assume that the line 114 be invoked only once per merch-card.